redash/serializers.py

```diff
@@ -100,6 +101,7 @@ def serialize_query(query, with_stats=False, with_visualizations=False, with_use
         'options': query.options,
         'version': query.version,
         'tags': query.tags or [],
+        'is_safe': ParameterizedQuery(query.query_text, query.options.get("parameters", [])).is_safe,
```

Comment:
> query.options.get("parameters", [])

Not the first time I see this. Maybe we should have a `parameters` property on the `Query` model?

Reply:
At first I thought 5bdab05, but then I thought 9d5e231. WDYT?